monitoring/monitorlib/scd_automated_testing/scd_injection_observation_api.py

```diff
@@ -0,0 +1,43 @@
+import enum
+from monitoring.monitorlib.typing import ImplicitDict
+from typing import List, Literal
+from monitoring.monitorlib.scd import Volume4D
+
+SCOPE_SCD_QUALIFIER_INJECT = 'utm.inject_test_data'
+
+class InjectionStatus(str, enum.Enum):
+    ''' A enum to hold results of flight processing as defined by the SCD test API '''
+    Planned = 'Planned'
+    Rejected = 'Rejected'
+    ConflictWithFlight = 'ConflictWithFlight'
+    Failed = 'Failed'
+
+class OperationalIntentTestInjection(ImplicitDict):
+    ''' A class to hold data for operational intent data that will be submitted to the SCD testing interface. '''
+    state: str
+    priorty: int = 0
+    volumes: List[Volume4D]
+    off_nominal_volumes: List[Volume4D]= []
+
+## Definitions around flight authorisation data that need to be submitted to the test injection interface
+   
+class FlightAuthorisationData(ImplicitDict):
+    '''A class to hold information about Flight Authorisation Test, for more information see https://github.com/interuss/dss/blob/master/interfaces/automated-testing/scd/scd.yaml#L317'''
```

Comment:
[No need to provide a link for this data type individually](https://github.com/interuss/dss/pull/657#discussion_r773361634)

Reply:
Thanks I have now removed the reference